resources/nl-core/nl-core-Vaccination-event.xml

```diff
@@ -37,6 +67,10 @@
         <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient" />
       </type>
     </element>
+    <element id="Immunization.doseQuantity">

```

Comment:
Saying that this should be UCUM is de facto a restriction. In addition, there was a need to relax this restriction to allow for tablets etc. as well here (#210). I think this definition should be removed or modified here, and in the zib layer, it should be made clear that this restriction only applies to the zib.

Reply:
Tablet can be expressed in mg instead of mL, which would still be UCUM. I can remove UCUM from the description, but it is a restriction anywat in the zib vaccination event profile. The zib profile has been modified (in this PR) to allow other UCUM units than mL. The reason this definition is added to the NL core profile and not the zib profiel is because the [zib defintion](https://zibs.nl/wiki/Vaccinatie-v4.0(2020NL)) clearly states the use mL. Is this correct of should we override the zib description in the zib profile?